.reuse/dep5

```diff
@@ -0,0 +1,11 @@
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: dfdutils
+Source: https://github.com/KhronosGroup/dfdutils
+
+Files: KHR/khr_df.h
+Copyright: 2015-2019 The Khronos Group Inc.
+License: MIT
```

Comment:
This is a Khronos header in the Data Formats repo. We should change its license to Apache 2.0.